cohorts/varcode_utils.py

```diff
@@ -11,7 +11,12 @@ def __init__(self, variant, variant_collection, patient):
 
     @property
     def variant_metadata(self):
-        return self.variant_collection.metadata[self.variant]
+        source_to_metadata = self.variant_collection.source_to_metadata_dict
```

Comment:
Actually in the `merge_variants` code before, we remapped this to be a `dict from source to variant-metadata`.

https://github.com/hammerlab/cohorts/blob/master/cohorts/cohort.py#L499-L506

That actually led to #81, but now it is the same behavior for single or merged variant collections